exchanges/order/order_test.go

```diff
@@ -2045,3 +2047,13 @@ func TestAdjustQuoteAmount(t *testing.T) {
 		t.Fatalf("received: '%v' but expected: '%v'", s.Amount, 5.22222222)
 	}
 }
+
+func TestSideUnmarshal(t *testing.T) {
+	t.Parallel()
+	var s Side
+	assert.Nil(t, s.UnmarshalJSON([]byte(`"SELL"`)), "Quoted valid side okay")
+	assert.Equal(t, Sell, s, "Correctly set order Side")
+	assert.ErrorIs(t, s.UnmarshalJSON([]byte(`"STEAL"`)), errUnrecognisedOrderSide, "Quoted invalid side errors")
```

Comment:
`errUnrecognisedOrderSide` doesn't exist

`ErrSideIsInvalid` ?

Reply:
Whoops. Sitting in my local branch unpushed 🤦 
Fixed 04e48924